src/manifest.json

```diff
@@ -3,10 +3,10 @@
   "version": "1.0.7",
   "description": "Profile and debug Liquid template on your Shopify store",
   "devtools_page": "devtools.html",
-  "permissions": ["storage", "identity", "activeTab"],
+  "permissions": ["storage", "identity", "activeTab", "webRequest"],
```

Comment:
Will adding this permission increase review times? I know we had this problem with other permissions, and why `activeTab` is preferred..

Instead of reading request headers, we could be including SFR status in Server Timing header which can be accessed via JS.

Reply:
Ah good point! I just copied the way Ubercorn did it. But it seemed a bit fragile (eg.: if you reload the plugin after the page, doesn't work).

Did you have a metric in mind, or just a bogus one?

Reply:
See my most recent comment, but I don't think we'll need this permission 🤞